### PR TITLE
Ensure resources are loaded through HTTPS

### PR DESCRIPTION
**Pull request by nottrobin**
- Make some ambiguous resources explicitly load over HTTPS
- Fix styling for sliders
- For domains that we *know* support HTTPS (like insights.ubuntu.com), make *all links* use HTTPS

QA
--

Click around the site, especially on touched pages, and make sure everything works and looks right.

### CODE REVIEW

File: static/js/global.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 var ai = [
   { url: "http://www.ubuntu.com", title:"Ubuntu" },
```
Comment (pmahnke) [inline static/js/global.js]: what about this link? will we be https everywhere? I assume we have to change it AFTER we move to https?  Perhaps we should file and issue for that.

also, we need up update this file in the [new repo](https://github.com/ubuntudesign/global-nav) as well.

Comment (nottrobin) [inline static/js/global.js]: Yep. We should merge this first, but I've made an issue for replacing www.ubuntu.com links:
https://github.com/ubuntudesign/www.ubuntu.com/issues/907

I'll add a comment now about updating global nav.

Comment (nottrobin) [inline static/js/global.js]: I also filed this PR: https://github.com/ubuntudesign/global-nav/pull/6

**Comment by nottrobin** [comment]
Okay I'm gonna merge this.

**Pull request merged**